src/libraries/System.Net.Primitives/src/System/Net/CookieException.cs

```diff
@@ -14,11 +14,11 @@ public CookieException() : base()
         {
         }
 
-        internal CookieException(string message) : base(message)
+        internal CookieException(string? message) : base(message)
         {
         }
 
-        internal CookieException(string message, Exception inner) : base(message, inner)
+        internal CookieException(string? message, Exception inner) : base(message, inner)
```

Comment:
the inner exception should also be nullable.

Reply:
Fixed.